src/js/components/CheckBoxGroup/CheckBoxGroup.js

```diff
@@ -0,0 +1,99 @@
+import React, { forwardRef, useContext, useMemo } from 'react';
+
+import { Box } from '../Box';
+import { CheckBox } from '../CheckBox';
+import { FormContext } from '../Form/FormContext';
+
+export const CheckBoxGroup = forwardRef(
+  (
+    {
+      checked: checkedProp,
```

Comment:
Now that I'm reading more detail here, I think this property should be `value`, to align more with Select and how the Form `value` will work.

Reply:
"CheckBoxGroup just joined the value party"